data.json

```diff
@@ -423,13 +423,13 @@
     "tags": ["clothing", "hacktoberfest", "stickers", "accessories", "expired"]
   },
   {
-    "name": "Gatsby",
+    "name": "Gatsby x Netlify",
     "difficulty": "hard",
     "description": "Get a pull request merged into Gatsby and receive a free shirt, socks, or stickers.",
-    "reference": "https://www.gatsbyjs.org/docs/contributor-swag/",
+    "reference": "https://www.gatsbyjs.com/contributing/contributor-swag/",
     "image": "https://i.imgur.com/e7GwaTN.png",
```

Comment:
Can you please also update the image with one from the store to match the new swags

Reply:
ok, i'm updating image URL